cloudbuild.yaml

```diff
@@ -9,19 +9,14 @@ steps:
         echo ASC_BROWSER_GA_TRACKING_ID=UA-$$ERB_GA_ID-3 >> build.env
         echo BipEx_BROWSER_GA_TRACKING_ID=UA-$$ERB_GA_ID-4 >> build.env
     secretEnv:
-      - "ERB_GA_ID"
+      - 'ERB_GA_ID'
 
-  - name: "gcr.io/cloud-builders/docker"
+  - name: 'gcr.io/kaniko-project/executor:latest'
```

Comment:
Yeah, the pros and cons are pretty much what you mentioned there. For build-time environment dependencies, I'm sometimes OK with the latest tag, whereas I probably wouldn't be happy with that for a production deployment where you need to know exactly what versions of things you're running.

In this case, i think it's a good idea to have used the latest tag during development, but now that we have a version that we know works, we can pin to that (this will also help keep a record of which kaniko built which exome results image).